elfi/visualization/visualization.py

```diff
@@ -397,3 +397,50 @@ def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
         axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_gp(gp, parameter_names, axes=None, resol=50, const=None, **kwargs):
+    """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+    Parameters
+    ----------
+    gp : GPyRegression, required
+    parameter_names : dict, required
+        Parameter names from model.parameters dict('parameter_name':(lower, upper), ... )`
+    axes : matplotlib.axes.Axes, optional
+
+    Returns
+    -------
+    axes : np.array of plt.Axes
+
+    """
+    n_plots = gp.input_dim
+    shape = (n_plots, n_plots)
+    axes, kwargs = _create_axes(axes, shape, **kwargs)
+
+    x_evidence = gp.X
+    y_evidence = gp.Y
+    if not const:
+        const = x_evidence[np.argmin(y_evidence), :]
+    bounds_min = x_evidence.min(axis=0)
+    bounds_max = x_evidence.max(axis=0)
```

Comment:
Could use the bounds given by the user, if the description is signature is correct. The `gp` object also has a `bounds` attribute.

Reply:
Which one is better in this case? `gp.bounds` or giving as a parameter from user?

Reply:
Default to `gp.bounds` but allow the user to override.

Reply:
`gp.bounds` is defined by user. It is a list of tuples which defines intervals on grid. What is the benefit of using `gp.bounds` instead of `x_evidence` which is taken from `gp.X`? I understand that we should give more freedom for users in case of defining models, but changing with `gp.bounds` is not clear for me.